src/mixins/java/org/spongepowered/common/mixin/api/minecraft/block/AbstractBlockMixin_API.java

```diff
@@ -57,4 +58,8 @@ public BlockSoundGroup soundGroup() {
         return (BlockSoundGroup) this.soundType;
     }
 
+    @Override
+    public boolean hasBlockEntity() {
+        return this instanceof EntityBlock;
```

Comment:
Honestly could make the same mixin in EntityBlockMixin_API and default this returning false?

Reply:
> Honestly could make the same mixin in EntityBlockMixin_API and default this returning false?

The `EntityBlock` interface does not extend any other interfaces, it is not related directly to `Block`. If simply create a mixin extending `BlockType` and implement this method with return `true`, it simply won't be called. The default method in `AbstractBlockMixin_API` is always called.